### ISSUE TITLE
Require: Support multiple fastq files in subfolder.

### ISSUE THREAD

**Issue by bakerwm** [smithlabcode/falco#36]
Falco `version 1.0.0` could not parse the correct output directory, when I work on multiple fastq files within subfolder.

Here is the directory structure:
```
$ tree demo/
demo/
├── test_rep1.fq.gz
└── test_rep2.fq.gz
```

Then Run Falco as following:

```
$ falco demo/*gz
[limits]        using default limit cutoffs (no file specified)
[adapters]      using default adapters (no file specified)
[contaminants]  using default contaminant list (no file specified)
[Fri Sep  2 08:46:59 2022] Started reading file demo/test_rep1.fq.gz
[Fri Sep  2 08:46:59 2022] reading file as gzipped FASTQ format
[running falco|===================================================|100%]
[Fri Sep  2 08:46:59 2022] Finished reading file
[Fri Sep  2 08:46:59 2022] Writing text report to demo/demo/test_rep1.fq.gz_fastqc_data.txt
[Fri Sep  2 08:46:59 2022] Writing HTML report to demo/demo/test_rep1.fq.gz_fastqc_report.html
Elapsed time for file demo/test_rep1.fq.gz: 0s
[limits]        using default limit cutoffs (no file specified)
[adapters]      using default adapters (no file specified)
[contaminants]  using default contaminant list (no file specified)
[Fri Sep  2 08:46:59 2022] Started reading file demo/test_rep2.fq.gz
[Fri Sep  2 08:46:59 2022] reading file as gzipped FASTQ format
[running falco|===================================================|100%]
[Fri Sep  2 08:46:59 2022] Finished reading file
[Fri Sep  2 08:46:59 2022] Writing text report to demo/demo/test_rep2.fq.gz_fastqc_data.txt
[Fri Sep  2 08:46:59 2022] Writing HTML report to demo/demo/test_rep2.fq.gz_fastqc_report.html
Elapsed time for file demo/test_rep2.fq.gz: 0s

$ falco --version
falco 1.0.0
```

The subfolder path is duplicated as follow: (so could not generate output files)
`Writing text report to demo/demo/test_rep1.fq.gz_fastqc_data.txt`，

**Comment by guilhermesena1** [comment]
Nice find! I can reproduce, looking into it now.

**Comment by guilhermesena1** [comment]
If you are working from a git clone, I pushed a fix here 0ce4305 if at all possible I'm interested whether that solves the problem for you too :)

**Comment by bakerwm** [comment]
> If you are working from a git clone, I pushed a fix here [0ce4305](https://github.com/smithlabcode/falco/commit/0ce43054f53f0ad662a40cde024f7fc307ab4cc0) if at all possible I'm interested whether that solves the problem for you too :)

Thanks for your quick response. Problem solved.
